### PR TITLE
fix terms

### PR DESCRIPTION
**Pull request by allanhirt**
As per the other stuff I've fixed.

### CODE REVIEW

**Comment by PRMerger6** [comment]
@allanhirt : Thanks for your contribution! The author, @MikeRayMSFT, has been notified to review your proposed change.

File: docs/relational-databases/system-dynamic-management-views/sys-dm-hadr-instance-node-map-transact-sql.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ dev_langs:
   - "TSQL"
 helpviewer_keywords: 
   - "Availability Groups [SQL Server], monitoring"
-  - "Availability Groups [SQL Server], WSFC clusters"
+  - "Availability Groups [SQL Server], WSFC"
```
Comment (MikeRayMSFT) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-hadr-instance-node-map-transact-sql.md]: Just need to verify if and how the change to this metadata field might change some functionality.

**Comment by MikeRayMSFT** [comment]
Thanks again @allanhirt - we're just checking one thing before proceeding.

**Comment by MikeRayMSFT** [comment]
#sign-off

**Comment by MikeRayMSFT** [comment]
@ allanhirt, I accepted this PR because we have previously spoken about the grammatical issues relating to WSFC. For the record, it stands for Windows Server Failover Cluster. Therefore, "WSFC cluster" is redundant. Also there is no product or feature called "Windows Server Failover Clustering." 

In the future, please provide a couple sentences of explanation when you create a PR.  In case I'm not the one who lands this request.

**Pull request merged**